src/components/select/CountryMultiSelectDropdown.module.scss

```diff
@@ -0,0 +1,64 @@
+@use 'theme';
+
+.dropdownWrapper {
+  width: calc(100% - 32px);
+}
+
+.styledInput {
+  div[class*='Amino__FloatLabelInput-module__inputPrefix'] {
+    justify-content: flex-start;
+    margin-right: 8px;
+    max-width: 50%;
+    flex-basis: content;
+  }
```

Comment:
Have you tried `:global` for this instead of searching regex for class
```suggestion
  :global(.Amino__FloatLabelInput-module__inputPrefix') {
    justify-content: flex-start;
    margin-right: 8px;
    max-width: 50%;
    flex-basis: content;
  }
```

Reply:
I have not tried, but the class is just a partial match because of the hashing so I don't think it would work. The has is after `...__inputPrefix_123asdasd23123sd`...

Reply:
<img width="329" alt="image" src="https://github.com/Zonos/amino/assets/50718218/1b72b1a3-8f62-42ff-9e5c-0d6875f5e4d7">

Reply:
I see, thanks for the info